app/controllers/users_controller.rb

```diff
@@ -30,13 +30,27 @@ def edit
   end
 
   def update
-    # require "pry"; binding.pry
     user = User.find(session[:user_id])
     user.update!(user_params)
     flash[:sucess] = "Changes Made to Profile Successfully"
     redirect_to '/profile'
   end
 
+  def edit_password
+    @user = User.find(session[:user_id])
```

Comment:
Added the user to change view so it did not interact with editing a user's information